language/en/common.php

```diff
@@ -44,6 +44,7 @@
 	'IDEA_TOPIC'            => 'Idea topic',
 	'IDEA_POSTER'           => 'Posted by %s', // Warning: submitted to db
 	'IDEAS_HOME'			=> 'Ideas Home',
+	'IDEAS_NOT_AVAILABLE'	=> 'You must configure Ideas properly to do this.',
```

Comment:
"Ideas is not available at this time." This is a user facing error, so it's shouldn't give away that the something is misconfigured.